etc/docker/Dockerfile.debian.bookworm

```diff
@@ -0,0 +1,44 @@
+FROM debian:bookworm
```

Comment:
This is a drive-by: bookworm has been released so I copied the `sid` `Dockerfile` to `bookworm` and updated it as needed.